example/workflows/sips.yml

```diff
@@ -207,6 +207,18 @@ IngestAndPublishGranule:
             - States.ALL
           IntervalSeconds: 2
           MaxAttempts: 3
+      Next: FilesToGranulesStep
+    FilesToGranulesStep:
```

Comment:
I'm not sure I follow the first part of your comment. This step will be tested as it is part of the IngestGranule integration tests. I agree on the second part, but fixing processing will probably require user intervention as well, as `cumulus_process` derived classes will need to be updated when the parent is.

Reply:
Really all I'm suggesting is we should remove the intermediate step from all but a test designed to test it (for legacy acceptance reasons).   We've already covered we'll do that in a future ticket so this can be resolved.